backpack-lint/src/main/java/net/skyscanner/backpack/lint/check/BpkComponentUsageDetector.kt

```diff
@@ -93,6 +96,17 @@ class BpkComponentUsageDetector : Detector(), SourceCodeScanner, XmlScanner {
     }
   }
 
+  override fun visitMethodCall(context: JavaContext, node: UCallExpression, method: PsiMethod) {
```

Comment:
some components get created via some static methods - this adds support for that